apis/identity/v1alpha1/iamaccesskey_types.go

```diff
@@ -37,6 +37,12 @@ type IAMAccessKeyParameters struct {
 	// +optional
 	IAMUsernameSelector *xpv1.Selector `json:"userNameSelector,omitempty"`
 
+	// Region can be used to supply an optional region.
+	// If no region is defined, aws-global region will be used
+	// which is the default region for IAM resources in the aws partition
+	// +optional
+	Region string `json:"region,omitempty"`
```

Comment:
```suggestion
	Region *string `json:"region,omitempty"`
```

Using pointer type is [recommended](https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#pointer-types-and-markers) for optional fields.